### PR TITLE
Fix SVG import issue

### PR DESCRIPTION
**Pull request by mmazanec22**
For some reason importing SVGs from USWDS into components here and using them in the app was resulting in only the URL being exported, and not the whole SVG as a ReactComponent.  Copying the USWDS images folder into this project and using local imports fixes the issue and removes the need for the sketchy inline svg component workaround.

### CODE REVIEW

**Review by bamader — APPROVED** [review]
WOOF this looked rough 😕 . Just so I understand, changing the import strategy and just using the SVG instead of an inline component is designed to make it load faster? There's no loss of style or information when doing this?

File: .eslintcache [hunk 1]
```diff
@@ -0,0 +1 @@
...ver used.","'HeadingLevel' is defined but never used.","'HTMLAttributes' is defined but never used.","'FieldSetProps' is defined but never used.","'FormFieldProps' is defined but never used.","'RadioButtonProps' is defined but never used.","'ProcessStepProps' is defined but never used.","'Column' is defined but never used.","'SingleCheckbox' is defined but never used.","'ReactEventHandler' is defined but never used.","'SortOrder' is defined but never used.","'ButtonProps' is defined but never used.","'AccordionItemProps' is defined but never used.","'Dispatch' is defined but never used.","'SetStateAction' is defined but never used.","'TabNav' is defined but never used.","'PropsWithChildren' is defined but never used.","'FieldSetStatusFunc' is defined but never used.","'NavLinkType' is defined but never used.","'InputHTMLAttributes' is defined but never used.","'InternalTabItemType' is defined but never used.","'Context' is defined but never used.","'TObjectDriller' is defined but never used.","'FieldStatusFunc' is defined but never used.","'FormHTMLAttributes' is defined but never used.","'Icon' is defined but never used.",["536"],["537"],[],"no-global-assign","no-unsafe-negation"]
```
Comment (JordanGuinn) [inline .eslintcache:1]: Why is this file included?

**Review by JordanGuinn — APPROVED** [review]
Nitpick of sorts - are we ever gonna need to come in here and update the USWDS SVGs accordingly?  Or are those static enough that it's not so much of a concern?

Approving regardless, just wondering out loud if it'd be worth making a note about that.

File: .eslintcache (hunk 1 above)
Comment (mmazanec22) [reply .eslintcache:1]: Leftover cruft!  Will add to gitignore

**Comment by mmazanec22** [comment]
@bamader the slow loading was bc the hacky workaround we were using was fetching and parsing each SVG instance individually in order to render it, bc of the weird React bug where only the file name was exported, and not the ReactComponent (I determined that it was an importing issue by looking at the js in the output of the build-- the local imports were handled differently than the ones from node modules for some reason).  So the only diff to the user will be faster load time.

@JordanGuinn I don't think the icons change too often.  It shouldn't be any more of a lift than any other library update-- just a matter of remembering to do it if we want to take advantage of new or different icons.  Will add a note in the readme.

FWIW I added all of the USWDS icons bc checking/updating individual files seemed like it would become an obnoxious maintenance task, and I imagine running "cp -r blablabla" once in a while is preferable.  There's no side effect either way for the user, so IMO we should go with whatever seems like the lowest maintenance lift.

**Comment by bamader** [comment]
Wow that sounds like it was some pretty gross digging to discover that import difference. Great find!

**Review by jlhogan — APPROVED** [review]
